### ISSUE TITLE
VmuStuff example is missing assets

### ISSUE THREAD

**Issue by Protofall** [Protofall/Crayon-Game-Framework#116]
When working on another issue, I noticed with Redream the VmuStuff demo gives the output "It failed with code 1". This meant when `crayon_savefile_load_eyecatch(&savefile_details, "Save/eyecatch3.bin");` returned 1 which means the file wasn't found.

It turns out the assets directory is near empty only including the fontsheet in pbm form. There is no romdisk present for some reason and no VMU LCD/icon/eyecatch binaries either.

Add in a romdisk called "sf_icon" to the root of the assets directory containing a "image.bin", "palette.bin", "eyecatch3.bin" and "LCD.bin". Once that is done this program should work again.
